src/fsharp/FSharp.Core/project.json

```diff
@@ -1,38 +1,38 @@
 {
   "dependencies": {
-    "Microsoft.NETCore.Platforms": "1.0.1-rc2-24027",
-    "Microsoft.NETCore.Runtime": "1.0.2-rc2-24027",
-    "System.Collections": "4.0.11-rc2-24027",
-    "System.Console": "4.0.0-rc2-24027",
-    "System.Diagnostics.Debug": "4.0.11-rc2-24027",
-    "System.Diagnostics.Tools": "4.0.1-rc2-24027",
-    "System.Globalization": "4.0.11-rc2-24027",
-    "System.IO": "4.1.0-rc2-24027",
-    "System.Linq": "4.1.0-rc2-24027",
-    "System.Linq.Expressions": "4.0.11-rc2-24027",
-    "System.Linq.Queryable": "4.0.1-rc2-24027",
-    "System.Net.Requests": "4.0.11-rc2-24027",
-    "System.Reflection": "4.1.0-rc2-24027",
-    "System.Reflection.Extensions": "4.0.1-rc2-24027",
-    "System.Resources.ResourceManager":"4.0.1-rc2-24027",
-    "System.Runtime": "4.1.0-rc2-24027",
-    "System.Runtime.Extensions": "4.1.0-rc2-24027",
-    "System.Runtime.Numerics": "4.0.1-rc2-24027",
-    "System.Text.RegularExpressions": "4.0.12-rc2-24027",
-    "System.Threading": "4.0.11-rc2-24027",
-    "System.Threading.Tasks": "4.0.11-rc2-24027",
-    "System.Threading.Tasks.Parallel": "4.0.1-rc2-24027",
-    "System.Threading.Thread": "4.0.0-rc2-24027",
-    "System.Threading.ThreadPool": "4.0.10-rc2-24027",
-    "System.Threading.Timer": "4.0.1-rc2-24027"
+    "Microsoft.NETCore.Platforms": "1.0.1",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2",
+    "System.Collections": "4.0.11",
+    "System.Console": "4.0.0",
+    "System.Diagnostics.Debug": "4.0.11",
+    "System.Diagnostics.Tools": "4.0.1",
+    "System.Globalization": "4.0.11",
+    "System.IO": "4.1.0",
+    "System.Linq": "4.1.0",
+    "System.Linq.Expressions": "4.1.0",
+    "System.Linq.Queryable": "4.0.1",
+    "System.Net.Requests": "4.0.11",
+    "System.Reflection": "4.1.0",
+    "System.Reflection.Extensions": "4.0.1",
+    "System.Resources.ResourceManager":"4.0.1",
+    "System.Runtime": "4.1.0",
+    "System.Runtime.Extensions": "4.1.0",
+    "System.Runtime.Numerics": "4.0.1",
+    "System.Text.RegularExpressions": "4.1.0",
+    "System.Threading": "4.0.11",
+    "System.Threading.Tasks": "4.0.11",
+    "System.Threading.Tasks.Parallel": "4.0.1",
+    "System.Threading.Thread": "4.0.0",
+    "System.Threading.ThreadPool": "4.0.10",
+    "System.Threading.Timer": "4.0.1"
   },
   "runtimes": {
     "win7-x86": { },
     "win7-x64": { },
-    "osx.10.10-x64": { },
+    "osx.10.11-x64": { },
     "ubuntu.14.04-x64": { }
   },
   "frameworks": {
-    "netstandard1.5": { }
+    "netstandard1.6": { }
```

Comment:
This isn't a concern right now, but eventually we'll want to find a way to lower the `netstandard` TFM for FSharp.Core as low as we can get it.